app/mailers/planning_application_mailer.rb

```diff
@@ -41,14 +41,13 @@ def invalidation_notice_mail(planning_application, host)
     )
   end
 
-  def receipt_notice_mail(planning_application, host, user)
-    @host = host
+  def receipt_notice_mail(planning_application)
     @planning_application = planning_application
 
     view_mail(
       NOTIFY_TEMPLATE_ID,
-      subject: "We have received your application",
-      to: user,
+      subject: "Lawful Development Certificate application received",
+      to: @planning_application.applicant_and_agent_email,
```

Comment:
oh nice, does this send two emails if this ends up being `["agent@example.com", "applicant@example.com"]`

Reply:
Thanks for bringing this up!

Reply:
Oh that's a shame an error isn't raised, good shout for checking. Doesn't seem anything obvious in the docs about sending to multiple addresses without iterating

Reply:
Oh that's a shame an error isn't raised, good shout for checking. Doesn't seem anything obvious in the docs about sending to multiple addresses without iterating